crates/matrix-sdk-ui/src/timeline/inner.rs

```diff
@@ -937,7 +933,7 @@ impl<P: RoomDataProvider> TimelineInner<P> {
                         let new_item = entry.with_kind(updated_item);
                         ObservableVectorEntry::set(&mut entry, new_item);
                     } else {
-                        trace!(event_id, transaction_id, "Profile already marked unavailable");
+                        debug!(event_id, transaction_id, "Profile already marked unavailable");
```

Comment:
It should actually *almost never* be triggered because we set all unavailable ones to pending before making the network request. This can only happen if the items were updated / new ones were added before that transformation, and before we lock the list again for updating the profiles.

Reply:
We can revert this if it turns out to be spammy (for a valid reason) in practice.

Reply:
I think having it in `debug` is better than `trace` :-).